src/examples/can/atsame54-xpro.go

```diff
@@ -0,0 +1,14 @@
+// +build atsame54_xpro
+
+package main
+
+import (
+	"machine"
+)
+
+func init() {
+	// Activate CAN Transceiver
+	stb := machine.CAN1_STANDBY
+	stb.Configure(machine.PinConfig{Mode: machine.PinOutput})
+	stb.Low()
```

Comment:
What does this do?

Reply:
Many CAN transceivers have Standby-mode.
In order to be able to communicate with CAN, it needs to be set to Normal-mode, so the port is manipulated.

* atsame54-xpro : ATA6561
    * https://www.microchip.com/wwwproducts/en/ata6561
* feather-m4-can : TCAN1051-Q1
    * https://www.ti.com/product/TCAN1051-Q1

Reply:
Perhaps adding an interface function such as `func (can *CAN) Standby(on bool) error` would be a good feature to clarify intent and also require less datasheet specific knowledge.

If the specific CAN transceiver does not support this mode, the function could return an error that was "Standby not supported" or something like that.

Reply:
Standby Pin was added to CAN.Configure(). This simplifies the transition from Standby-mode to Normal-mode. Thanks @deadprogram .

On the other hand, I don't think Standby(bool) is necessary for the CAN function at the moment, because I haven't created a side to stop CAN. I will PR it again in the future when I want to stop CAN. A typical use of Standby is to reduce the power consumption when the microcontroller is put to sleep.

Reply:
OK make sense. Thanks!